frontend/src/settings/beta/ThemeTab.tsx

```diff
@@ -0,0 +1,152 @@
+import { Box, Button, Card, CardContent, CardHeader, Divider, Grid, Stack, Tab, Tabs, Theme } from '@mui/material'
+import { styled, ThemeProvider } from '@mui/material/styles'
+
+import { betaLightTheme, darkTheme, earthTheme, lightTheme, sunsetTheme } from '../../../src/theme'
+import { User } from '../../../types/types'
+
+interface ThemeTabProps {
+  user: User
+}
+
+interface ThemeKey {
+  name: string
+  theme: Theme
+}
+
+// TODO - determine how we are storing user themes
+
+export default function ThemeTab({ user }: ThemeTabProps) {
+  const themes: ThemeKey[] = [
```

Comment:
This could be defined outside of the component, so it's not recreated on every render